features/gesplitst-zakelijk-recht.feature

```diff
@@ -0,0 +1,429 @@
+                                "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden/1002195699"
+                            }
+                        }
+                    }
+                ]
+            },
+            "gesplitstZakelijkRecht": "erfpachter"
+        }
+        """
+
+Rule: parameter type bij zakelijkgerechtigden werkt ook op gesplitstZakelijkRecht
+
+    Scenario: opvragen van de zakelijk gerechtigden met type van ongesplitst recht
+        Als zakelijk gerechtigde wordt gevraagd met "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=eigenaar"
+        Dan is het antwoord:
+        """
+        {
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=eigenaar"
+                }
+            },
+            "_embedded": {
+                "zakelijkGerechtigden": [
+                    {
+                        "identificatie": "1002195699",
+                        "type": "eigenaar",
+                        "persoon": {
+                            "omschrijving": "Anne Janssens"
+                        },
+                        "_links": {
+                            "self": {
+                                "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden/1002195699"
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+        """
+
+    Scenario: opvragen van de zakelijk gerechtigden met type van gesplitst recht
+        Als zakelijk gerechtigde wordt gevraagd met "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=erfpachter"
+        Dan is het antwoord:
+        """
+        {
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=erfpachter"
+                },
+                "kadastraalOnroerendeZakenUitSplitsing": {
+                    "href": "/kadastraalonroerendezaken?kadastraleAanduiding=Groningen+P+1399&inclusiefKadastraalOnroerendeZakenUitSplitsing=true"
+                },
+                "kadastraalOnroerendeZakenUitSplitsingInclusiefZakelijkGerechtigden": {
+                    "href": "/kadastraalonroerendezaken?expand=zakelijkGerechtigden&kadastraleAanduiding=Groningen+P+1399&inclusiefKadastraalOnroerendeZakenUitSplitsing=true"
+                }
+            },
+            "_embedded": {},
+            "gesplitstZakelijkRecht": "erfpachter"
+        }
+        """
+
+Rule: bij een onroerende zaak met gesplitst zakelijk recht wordt het gesplitste recht getoond in gesplitstZakelijkRecht
+
+    Scenario: opvragen van een perceel met gesplitst zakelijk recht
```

Comment:
Waarom is in deze twee scenario's gesplitstZakelijkRecht opgenomen in de fields parameter? De gesplitstZakelijkRecht property wordt toch altijd gezet als je zakelijkgerechtigden opvraagt en er zij zakelijkgerechtigden uit splitsing?

Reply:
gesplitstZakelijkRecht wordt onafhankelijk geleverd van wat in fields staat bij /zakelijkgerechtigden. gesplitstZakelijkRecht maakt namelijk geen deel uit van de zakelijkgerechtigden resource. Met de fields parameter kan je alleen de inhoud van de resource beïnvloeden

gesplitstZakelijkRecht is wel toegevoegd aan de kadastraalonroerendezaken resource. we hebben besloten deze altijd te leveren, zie #932, dus in de fields parameter is gesplitstZakelijkRecht niet meer nodig. Ik heb hier fields gebruikt om de response te beperken, anders is het antwoord in de feature te groot en worden er veel zaken getoond die niet relevant zijn voor wat in deze feature geïllustreerd wordt.